content/docs/operating/configuration.md

```diff
@@ -323,6 +323,9 @@ Kubernetes API
 * `__meta_kubernetes_service_annotation_<annotationname>`: each service
 annotation from the Kubernetes API
 
+In addition, the `instance` label for node metrics will be set to the node name
```

Comment:
Also remove the `__meta_kubernetes_node` label documentation above, since that no longer exists?